flight_phase_files/HALO_RF11_20200209_info.yaml

```diff
@@ -22,68 +26,127 @@ remarks:
 - At the 1st Ron Brown flyover a dropsonde was launched.
 - Mostly shallow sugar clouds during the flight.
 - Two cold pools entered the circle during the 2nd circle set.
+- At the beginning of the leg inbound from NTAS, a diversion due to traffic was needed.
 segments:
-- kind: circle
-  name: c1
-  start: 2020-02-09 09:36:08
-  end: 2020-02-09 10:27:24
-- kind: circle_break
-  name: cb1to2
-  start: 2020-02-09 10:27:24
-  end: 2020-02-09 10:50:11
-- kind: circle
-  name: c2
-  start: 2020-02-09 10:50:11
-  end: 2020-02-09 11:42:45
-- kind: circle_break
-  name: cb2to3
-  start: 2020-02-09 11:42:45
-  end: 2020-02-09 11:59:21
-- kind: circle
-  name: c3
-  start: 2020-02-09 11:59:21
-  end: 2020-02-09 12:54:05
-- kind: straight_leg
-  name: leg1_outbound_NTAS
-  start: 2020-02-09 12:59:59
-  end: 2020-02-09 13:17:03
-- kind: radar_calibration_wiggle
-  name: radar_cal_wiggle1
-  start: 2020-02-09 13:17:03
-  end: 2020-02-09 13:19:42
-- kind: radar_calibration_tilted
-  name: radar_cal_tilted
+- kinds: [circle]
+  name: circle 1
+  start: 2020-02-09 09:35:28
+  end: 2020-02-09 10:32:48
+  segment_id: HALO-0209_c1
+  irregularities: [begin is 41s before first launch due to roll angle at start]
+  good_dropsondes: 11
+- kinds: [circle_break]
+  name: circle break 1/2
+  start: 2020-02-09 10:32:24
+  end: 2020-02-09 10:49:12
+  segment_id: HALO-0209_cb1
+  irregularities: []
+- kinds: [circle]
+  name: circle 2 
+  start: 2020-02-09 10:49:12
+  end: 2020-02-09 11:46:44
```

Comment:
Also here, it seems like you could add one second at the end of the circle 2.
```suggestion
  end: 2020-02-09 11:46:45
```

Reply:
That would create a tiny overlap:
![grafik](https://user-images.githubusercontent.com/6574622/84156599-fc55f080-aa69-11ea-83de-38ef60c994ff.png)
I tend to go for the 44 in stead.